### DISCUSSION TITLE
page.search

### DISCUSSION THREAD

**Discussion by yonglee7015** [jsvine/pdfplumber#805] [Ask for help with specific PDFs]
page.search  can not search sentence with \n break.  
for example, 
search Description (Long Description) not works, but search  Description (Long, it works.

![image](https://user-images.githubusercontent.com/59268746/216506987-c8164377-f46e-4b56-bc5b-fdaddaea0936.png)

**Comment by jsvine** [comment]
If you expect newlines in the text you're searching, you can use the regular expression token `\s`, which represents whitespace (i.e., space characters, tabs, newlines). For your example: 

```python
page.search(r"Description\s+\(Long\s+Description\)")
```

Does that work for you?

**Reply by yonglee7015** [reply]
I just wonder if it also can search Description (Long Description) without \s . sometimes it happens without new line.

**Reply by jsvine** [reply]
`\s` will match any of the following: space, newline, tab, and a few other similar characters. See here for details: https://docs.python.org/3/library/re.html#re-syntax

**Reply by yonglee7015** [reply]
Really appreciate for your help.
